components/members/Card.vue

```diff
@@ -10,13 +10,13 @@
                 ? props.member.avatar_url
                 : avatarPlaceholder
             "
-            class="rounded-xl w-16 md:w-full"
+            class="rounded-md w-16 md:w-full"
             loading="lazy"
           />
         </div>
         <div>
           <h3
-            class="text-base md:text-lg flex-nowrap lg:text-3xl font-semibold mb-1 md:mb-2"
+            class="title text-base md:text-lg flex-nowrap lg:text-2xl mb-1 md:mb-2"
```

Comment:
you dont need this class here, the font is already in sans. as for the font-weight, the tailwind `font-medium` class serves the purpose of font-weight 500